README.md

```diff
@@ -58,10 +58,18 @@ the ixbrl-viewer.
    iXBRLViewerPlugin/viewer/dist directory.
 
 [ixbrlviewer-github]: https://github.com/Workiva/ixbrl-viewer
+[CDN]: https://cdn-prod.wdesk.org/ixbrl-viewer/1.0.0/ixbrlviewer.js
 [ixbrlviewer-github-releases]: https://github.com/Workiva/ixbrl-viewer/releases/tag/0.1.58
 [arelle-git]: https://github.com/Arelle/Arelle
 [arelle-download]: http://arelle.org/pub
 
+# Javascript Versioning
+
+The ixbrl-viewer plugin embeds processed XBRL metadata in the HTML that has a specific format read 
+by the JavaScript. The metadata produced by version `MAJOR.MINOR.PATCH` is supported by any version
+of the JavaScript version `MAJOR.MINOR` and >= `PATCH`.
```

Comment:
It seems like it should be this?
> of the JavaScript version `MAJOR` and >= `MINOR.PATCH`.

Well, in English:
1. if a major version bump, then the JSON format is broken, and the new js won't necessarily work with older versions of the generated JSON
2. if a minor version bump, then something was added to the JSON format, so if you generate with some minor version, you have to use at least that minor version for the js.